boards/arm/ev11l78a/doc/index.rst

```diff
@@ -0,0 +1,81 @@
+.. _ev11l78a:
+
+UPD301C Basic Sink Application Example
+######################################
+
+Overview
+********
+
+The UPD301C Basic Sink Application Example Evaluation Kit (EV11L78A)
+is a low-cost evaluation platform for Microchip's UPD301C Standalone
+Programmable USB Power Delivery (PD) Controller. This RoHS-compliant
+evaluation platform comes in a small form factor and adheres to the
+USB Type-C™ Connector Specification and USB PD 3.0 specification.
+
+.. image:: img/ev11l78a.jpg
+     :align: center
+     :alt: EV11L78A
+
+   UPD301C Basic Sink Application Example (Credit: Micropchip Technology)
```

Comment:
```
/runner/_work/zephyr/zephyr/doc/_build/src/boards/arm/ev11l78a/doc/index.rst:15: ERROR: Error in "image" directive:
no content permitted.

.. image:: ../../../../../../../boards/arm/ev11l78a/doc/img/ev11l78a.jpg
     :align: center
     :alt: EV11L78A

   UPD301C Basic Sink Application Example (Credit: Micropchip Technology)
```

I think this line is not indented correctly? The error sound funny

Reply:
indentation was correct, but only figure:: accept contents (the description of the figure), and this originally used image::